package.json

```diff
@@ -28,26 +28,21 @@
     "test"
   ],
   "devDependencies": {
-    "eslint": "^8.31.0",
-    "eslint-config-standard": "^17.0.0",
-    "eslint-plugin-import": "^2.26.0",
-    "eslint-plugin-n": "^15.6.0",
-    "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-promise": "^6.1.1",
-    "pino": "^8.8.0",
-    "pino-socket": "^7.3.0",
-    "pre-commit": "^1.2.2",
-    "tap": "^16.3.2"
+    "eslint": "^9.39.1",
+    "lefthook": "^2.0.9",
```

Comment:
I changed `pre-commit` to `lefthook` due to two vulnerabilities in `pre-commit`.